meta/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+galaxy_info:
+  role_name: miniflux
+  author: ajnasz
+  description: Setup miniflux, minimalist and opinionated feed reader
+  license: BSD
+  min_ansible_version: '1.9'
+  standalone: true
+  platforms:
+    - name: EL
+      versions:
+        - all
+    - name: Ubuntu
+      versions:
+        - jammy
+        - focal
+        - bionic
```

Comment:
I'm not sure Ubuntu must be listed specifically, while others like Debian is not.